docs/source/logging/logging.md

```diff
@@ -19,19 +23,50 @@ log.info("Send information")
 ```
 
 ```{note}
-The name of a logger corresponds to a key in the `loggers`  section in `logging.yml` (e.g. `kedro.io`). See [Python's logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information.
+The name of a logger corresponds to a key in the `loggers`  section in `logging.yml` (e.g. `kedro`). See [Python's logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information.
+```
+
+You can take advantage of rich's [console markup](https://rich.readthedocs.io/en/stable/markup.html) when enabled in your logging calls: 
+```python
+log.error("[bold red blink]Important error message![/]", extra={"markup": True})
 ```
 
-## Logging for `anyconfig`
+## Default framework-side logging configuration
+
+Kedro's [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/config/logging.yml) defines a handler called `rich` that uses the [Rich logging handler](https://rich.readthedocs.io/en/stable/logging.html) to format messages and handle exceptions.
+
+By default, Python only shows logging messages at level `WARNING` and above. Kedro's logging configuration specifies that `INFO` level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a `kedro run`.
+
+## Project-side logging configuration
+
+In addition to the `rich` handler defined in Kedro's framework, the [project-side `conf/base/logging.yml`](https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/conf/base/logging.yml) defines three further logging handlers:
+* `console`: show logs on standard output (typically your terminal screen) without any rich formatting
+* `info_file_handler`: write logs of level `INFO` and above to `logs/info.log`
+* `error_file_handler`: write logs of level `ERROR` and above to `logs/error.log`
+
+The logging handlers that are actually used by default are `rich`, `info_file_handler` and `error_file_handler`.
+
+The project-side logging configuration also ensures that [logs emitted from your project's logger](#use-logging) should be shown if they are `INFO` level or above (as opposed to the Python default of `WARNING`). 
+
+We now give some common examples of how you might like to change your project's logging configuration.
+
+### Disable file-based logging
```

Comment:
Perhaps add a sentence or two about why you'd want to disable file-based logging, e.g. for the Databricks read-only stuff etc..